edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/Startup.cs

```diff
@@ -127,6 +127,10 @@ IContainer BuildContainer(IServiceCollection services)
             int scopeCacheRefreshRateSecs = this.Configuration.GetValue("DeviceScopeCacheRefreshRateSecs", 3600);
             TimeSpan scopeCacheRefreshRate = TimeSpan.FromSeconds(scopeCacheRefreshRateSecs);
 
+            int cloudConnectionIdleTimeoutSecs = this.Configuration.GetValue("CloudConnectionIdleTimeoutSecs", 3600);
+            TimeSpan cloudConnectionIdleTimeout = TimeSpan.FromSeconds(cloudConnectionIdleTimeoutSecs);
+            bool closeCloudConnectionOnIdleTimeout = this.Configuration.GetValue("CloseCloudConnectionOnIdleTimeout", true);
```

Comment:
Why would someone change this setting to false?

Reply:
If you don't care about having additional cloud connections.. then you might want to set it to false.

Reply:
Per our offline conversation, it's mostly there for debugging.